### PR TITLE
Refactors

### PR DESCRIPTION
**Pull request by sagikazarmark**
## Overview

This PR refactors a number of things in OpenMeter:

- Removed unnecessary slog initialization
- Relocated kafka config mapping
- Relocated ksqldb config mapping

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -186,22 +170,7 @@ func main() {
...
-	}
-	if config.Ingest.Kafka.SecurityProtocol != "" {
-		kafkaConfig["sasl.mechanism"] = config.Ingest.Kafka.SaslMechanisms
-	}
-	if config.Ingest.Kafka.SecurityProtocol != "" {
-		kafkaConfig["sasl.username"] = config.Ingest.Kafka.SaslUsername
-	}
-	if config.Ingest.Kafka.SecurityProtocol != "" {
-		kafkaConfig["sasl.password"] = config.Ingest.Kafka.SaslPassword
-	}
```
Comment (sagikazarmark) [inline main.go:203]: @hekike this looks like a copy-paste error. Can you please confirm if it is?

Comment (hekike) [reply main.go:203]: ouch, yes it is

Comment (hekike) [reply main.go:203]: do you plan to fix it in this PR or should I open a fix?

Comment (sagikazarmark) [reply main.go:203]: Yes, I'll take care of it.

**Review by hekike — APPROVED** [review]

**Pull request merged**